x/jwt_helper.go

```diff
@@ -56,19 +56,22 @@ func ExtractUserName(jwtToken string) (string, error) {
 	return userId, nil
 }
 
-func ExtractJWTNamespace(ctx context.Context) (uint64, error) {
-	jwtString, err := ExtractJwt(ctx)
-	if err != nil {
-		return 0, err
-	}
-	claims, err := ParseJWT(jwtString)
+func ExtractNamespaceFromJwt(jwtToken string) (uint64, error) {
+	claims, err := ParseJWT(jwtToken)
 	if err != nil {
 		return 0, err
 	}
-
 	namespace, ok := claims["namespace"].(float64)
 	if !ok {
 		return 0, errors.Errorf("namespace in claims is not valid:%v", namespace)
 	}
 	return uint64(namespace), nil
 }
+
+func ExtractJWTNamespace(ctx context.Context) (uint64, error) {
+	jwtString, err := ExtractJwt(ctx)
+	if err != nil {
+		return 0, err
```

Comment:
I think this error should be wrapped so that some context other than the JWT-focused error message is returned, e.g., `return 0, errors.Wrap(err, "extracting namespace from JWT")`